packages/nimble-components/src/button/specs/button-visual-states.md

```diff
@@ -0,0 +1,45 @@
+# Nimble-Button New Visual Designs Implementation
+
+## Problem Statement
+
+The Visual Design spec for Nimble buttons now includes primary button states `primary` and `primary-accent` that aren't implemented in the nimble-button component yet.
+
+## Links To Relevant Work Items and Reference Material
+
+[Nimble Issue #663: Adopt nimble-button visual designs for states primary, quiet, and loud](https://github.com/ni/nimble/issues/663)
+
+[Figma Spec](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1295-77205)
+
+## Implementation / Design
+
+There are three possible values for `appearance-variant` on a button: `default`, `primary`, and `primary-accent`, each with its own color scheme based on the UI theme.
+`primary` and `primary-accent` states of buttons are primary buttons, which can be used together up to 3 times on a page.
+
+The `primary` button can be used when there is a conflict with color and its context.
```

Comment:
I think by the time we get to capturing these recommendations in user-facing documentation, we'll want to be a lot more assertive (I realize you got these from Brandon's comments in Figma). Based on the discussion yesterday I think we should start with something more like this:

```suggestion
The `primary` button should be the default choice when one button needs to be emphasized.
```